data/wordlists/default_userpass_for_services_unhash.txt

```diff
@@ -1785,3 +1785,36 @@ AURORA$JIS$UTILITY$
 wlcsystem wlcsystem
 news 
 CPRM 
+root xc3511
+root vizxv
+root 888888
+root xmhdipc
+root juantech
+root 123456
+root 54321
+root 1111
+root 666666
+root klv123
+service service
+admin1 password
+666666 666666
+888888 888888
+ubnt ubnt
+root klv1234
+root Zte521
+root hi3518
+root jvbzd
+root anko
+root zlxx.
+root 7ujMko0vizxv
+root 7ujMko0admin
+root ikwb
+root dreambox
+root user
+root realtek
+root 00000000
+admin 1111111
+admin 54321
+admin 7ujMko0admin
+admin meinsm
+mother fucker
```

Comment:
1817 -  PacketIP VoIP Phone, 1818 - Dahua IP Camera, 1819 - Mobotix IP camera, 1820 - dunno, the mirai hacker had some interest in puting it in. 

All the lists were scraped from the mirai source code from github. 
For more info on those passwords, see: https://krebsonsecurity.com/2016/10/who-makes-the-iot-things-under-attack/

I created the mirai file just so anyone can test their devices against a standard mirai brute scanner and see if they would be compromised. Updated unhash ones, since i believe the unhash lists are most comprehensive for default services. We do need a "anotated" repository for such stuff, but that's for another project.  :)

Sincerely,
Tonimir